src/app/business/common/components/detail/components/metadata.js

```diff
@@ -170,15 +173,15 @@ OwnerMetadata.defaultProps = {
 };
 
 export const MetadataMetadata = ({metadata}) => {
-    const keys = Object.keys(metadata);
+    const keys = Object.keys(metadata).sort();
 
     return (keys.length ? (
         <>
             <SingleMetadata label="Metadata" />
-            {keys.sort().map((key) => (
-                <SingleMetadata labelClassName={metadataDt} valueClassName={metadataDd} key={key} label={key} value={metadata[key]} />))}
+            {keys.map((key) => (
+                <SingleMetadata labelClassName={metadataDt} valueClassName={metadataDd} key={key} label={`- ${key}`} value={metadata[key]} />))}
```

Comment:
Including the `-` like this works visually, but it introduces an unwanted behavior: When selecting the key, you'll also select the `-`. I guess this will also be picked up by screen readers.

One solution could be to have the `metadataDt ` updated with:
```css
 &:before {
  content: '- ';
}
```

Of course, the best solution would be to have an actual list, but in order to keep this short, the above solution should be enough.

Reply:
You're right, it will be better this way